### DISCUSSION TITLE
0.70.8

### DISCUSSION THREAD

**Discussion by benhillis** [microsoft/WSL#9116] [General]
* Fix various issues with plan9 logging
* Do not print any status updates when exporting to stdout
* Ensure /tmp/.X11-unix is not cleared by systemd [GH 9038]
* Add HCS_E_CONNECTION_TIMEOUT to g_commonErrors
* Add --enable-debug when building musl for better stacks
* Retry block device operations on ENXIO
* Fix race condition in the plan9 server causing ECONNREFUSED
* Add SharedAccess host firewall rule
* Update Microsoft.WSLg to version 1.0.47
    * WSLGd: fix relaunch weston with env
    * WSLG: update mariner to 2.0 official release
    * WSLGd: independently monitor child process for font monitor
    * WSLGd: add timestamp to log
    * WSLGd: redirect FreeRDP log to a file
    * WSLGd: Raise the NOFILE limit
    * WSLGd: misc updates (allow to load X cursor from user distro and etc.)
    * WSLGd: Include missing array header
    * system-distro: fix build error at mesa after mariner update
    * rdp shell: workaround get_position crash
    * rdp shell: fix missing va_end() call
    * rdp shell: make wslpath optional for obtaining windows path
    * rdp-backend: fix overactive assert
Update Kernel to version 5.15.74.2
    - Update to stable kernel version v5.15.74
    - Enable the limit option (CONFIG_NETFILTER_XT_MATCH_LIMIT) for iptables
    - Include fix for hvsocket read() hang

<hr /><em>This discussion was created from the release <a href='https://github.com/microsoft/WSL/releases/tag/'>0.70.8</a>.</em>

**Comment by tellisnz-shift** [comment]
Roughly how quickly will pre releases become releases/deployed to the store for update via `wsl --update`?

If I wanted to update my local WSL2 currently installed from the store using the asset attached to the release, where can I find the info on how to do that? Can I then later have the normal update cycle of `wsl --update` take over?
